update.go

```diff
@@ -53,9 +53,14 @@ func (c *Client) updateReleaseFromTarball(ctx context.Context, chartPath, namesp
 }
 
 func (options UpdateOptions) configure(action *action.Upgrade, namespace string) {
+	//if options.Timeout == 0 {
```

Comment:
We can delete those comments?

Reply:
Good catch. I think we should keep them to be sure there is a timeout in place.